test/sanity/basics/numbers/extensionMethods.wtest

```diff
@@ -0,0 +1,273 @@
+/**
+ * Tests for Wollok numbers => extensions methods
```

Comment:
Can we rename this file?

Reply:
Which name would you prefer?

Reply:
The problem with the name is that is implementation specific. In ts there are no extension methods, so we would not know what to put where. We could just group this tests with the other number tests. If that becomes too large, we could split them by general purpose (like "arithmetic") or grammar (like "operators").

Reply:
ok, lets use `operators.wtest`